app/ViewModels/Concerns/Transaction/HasType.php

```diff
@@ -63,6 +63,12 @@ public function isUsernameResignation(): bool
         return $this->type->isUsernameResignation();
     }
 
+    // TODO: implement method correctly - https://app.clickup.com/t/86dur8fj6
+    public function isContract(): bool
```

Comment:
🤔 not sure `isContract` makes sense as it's for every EVM call that we need to show this which includes non-contract calls as well for transfers

Reply:
Okies. I added it as a placeholder for the task. Do you think `isEvm` would be more appropriate?